lib/api/src/rest/schema.rs

```diff
@@ -19,7 +20,7 @@ pub type MultiDenseVector = Vec<DenseVector>;
 #[serde(untagged, rename_all = "snake_case")]
 pub enum Vector {
     Dense(DenseVector),
-    Sparse(sparse::common::sparse_vector::SparseVector),
+    Sparse(sparse::common::sparse_vector::SparseVector<VectorElementType>),
```

Comment:
I don't think we want to make `Vector` itself generic against datatype. It would require too much changes.